### DISCUSSION TITLE
Multi core simulation

### DISCUSSION THREAD

**Discussion by amalrasas** [ChampSim/ChampSim#453] [Q&A]
Hi, i was wondering if i can use these traces that you provided https://dpc3.compas.cs.stonybrook.edu/champsim-traces/speccpu/
For multi-core simulation or is there specific traces that i should use for multi-core?

**Comment by ngober** [comment]
Yes, the simulator currently does not support multithreaded programs. The only way to run a multicore simulation is to run a distinct program (that is, a distinct trace) on each core.
